pkg/security/podsecuritypolicy/selinux/mustrunas.go

```diff
@@ -74,3 +81,128 @@ func (s *mustRunAs) Validate(fldPath *field.Path, _ *api.Pod, _ *api.Container,
 
 	return allErrs
 }
+
+func equalLevels(expectedLevel, actualLevel string) (bool, error) {
+	if expectedLevel == actualLevel {
+		return true, nil
+	}
+
+	// Level format: https://selinuxproject.org/page/MLSStatements#level
+
+	// "s0:c6,c0" => [ "s0", "c6,c0" ]
+	expectedParts := strings.SplitN(expectedLevel, ":", 2)
+	actualParts := strings.SplitN(actualLevel, ":", 2)
+
+	// "s0-s0" => [ "s0" ]
+	expectedSensitivity, err := canonicalizeSensitivity(expectedParts[0])
+	if err != nil {
+		return false, err
+	}
+
+	actualSensitivity, err := canonicalizeSensitivity(actualParts[0])
+	if err != nil {
+		return false, err
+	}
+
+	if !reflect.DeepEqual(expectedSensitivity, actualSensitivity) {
```

Comment:
Do we really want to use DeepEqual here, this uses reflection.

You can just use `==` for arrays in golang.

See:
https://golang.org/ref/spec#Comparison_operators

"Array values are comparable if values of the array element type are comparable. Two array values are equal if their corresponding elements are equal."

Reply:
Unfortunately, we can't use `==` here because we're comparing slices and not arrays. These slices are came from invocation of `strings.Split()` and may be differ in length.

Reply:
I would probably prefer a custom equality function over reflect.DeepEqual.  You are checking that two string arrays are equal, better to have an explicit method `stringSliceEqual` or something

Reply:
Added in the last update.